composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/dags/example_dag_test.py

```diff
@@ -13,20 +13,20 @@
 # limitations under the License.
 
 from airflow import models
-import airflow.utils.db
 import internal_unit_testing
 import pytest
 
 # user should substitute their project ID
 PROJECT_ID = 'your-project-id'
 
 
-@pytest.fixture(autouse=True, scope="module")
-def initalize_airflow_database():
-    airflow.utils.db.initdb()
+@pytest.fixture(autouse=True, scope="function")
+def set_variables():
```

Comment:
Thoughts on moving the `airflow_database` fixture here instead? That way we'd know it runs before the function to set variables. Is this why `airflow_database` is a `session` scope fixture: so it runs before function scope fixtures?

Reply:
Yes - this 100% makes sense and I'm making that change now. Good catch